src/components/widget/legend/component.js

```diff
@@ -5,19 +5,23 @@ import classnames from 'classnames';
 
 import styles from './style.module.scss';
 
-const Legend = ({ groups, direction, variant = 'rect' }) => (
+const Legend = ({ type, groups, direction, variant = 'rect' }) => (
   <div className={classnames(styles.widget_legend, { [styles.vertical]: direction === 'vertical' })}>
     {Object.keys(groups).map(g => (
       <div key={g} className={styles.widget_legend_group}>
-        <ul className={styles.widget_legend_list}>
+        <ul className={classnames(styles.widget_legend_list, styles[`_${type}`])}>
           {groups[g].map(item => (
             <li
               key={`item-${item.color}`}
-              className={classnames(styles.widget_legend_list_item, styles[`_${variant}`])}
+              className={classnames(styles.widget_legend_list_item,
+                styles[`_${variant}`],
+                styles[`_${type}`])
+              }
             >
               <svg height="12" width="12">
                 <rect
-                  className={classnames(styles.item, { [styles.net]: (item.value).toLowerCase() === 'net result' })}
+                  className={classnames(styles.item,
+                    { [styles.net]: (item.value).toLowerCase() === 'net result' })}
```

Comment:
I think having net result string hardcoded here reduces component flexibility and make it more difficult to maintain. Isn't it possible to add this style by using variant or type?

Reply:
Instead of using the use case (net result) as selector it would be better use a modifier o selector that describes how the style will modify appearance. Like `._rounded` or something similar that can be considered a variant?